src/WikiService.Data/RubricRepository.cs

```diff
@@ -112,5 +88,34 @@ public async Task<bool> DoesRubricNameExistAsync(Guid? rubricParentId, string na
     {
       return await _provider.Rubrics.AnyAsync(p => p.ParentId == rubricParentId && p.Name == nameRubric);
     }
+
+    private IQueryable<DbRubric> FindRubricChild(IQueryable<DbRubric> dbRubric)
+    {
+      foreach (DbRubric topRubric in dbRubric.ToList())
+      {
+        foreach (DbRubric rubric in _provider.Rubrics.AsEnumerable())
+        {
+          if (rubric.ParentId == topRubric.Id)
+          {
+            topRubric.HasChild = true;
+            break;
+          }
+        }
```

Comment:
try to write it with Any instead of this foreach
if (await _provider.Rubrics.AnyAsync(x => x.ParentId == topRubricId))
{...}